Classes/Editor/MultiEditor/MultiEditorViewController.swift

```diff
@@ -147,6 +147,8 @@ class MultiEditorViewController: UIViewController {
                 self?.clipsController.removeDraggingClip()
             }
             load(childViewController: editor, into: editorContainer)
+            editorContainer.layoutIfNeeded()
```

Comment:
@bjtitus could you take a look at the fix, I saw you being active in the repository?

The fix is not what I hoped for but I couldn't find another way to force layout updates then we load a new image after the old ones were deleted.

Let me know if you don't have time or if anything in the PR is missing.

Thank you! 🙏

Reply:
@bjtitus Do you think you'll have time to take a look at the PR or maybe there're some other people familiar with this repo? Thanks!